core/src/main/java/org/elasticsearch/search/aggregations/AbstractAggregationBuilder.java

```diff
@@ -116,6 +116,11 @@ public AB setMetaData(Map<String, Object> metaData) {
     }
 
     @Override
+    public Map<String, Object> getMetaData() {
+        return metaData;
```

Comment:
Sure, I can add that.  Do we have a policy that we don't want to allow side-effects on builders, and force a get/set cycle rather than modifying in-place?